imports/plugins/included/payments-stripe/server/methods/stripe.js

```diff
@@ -97,22 +97,43 @@ Meteor.methods({
     });
     check(paymentData, {
       total: String,
-      currency: String
+      currency: String,
+      shopId: String
     });
 
-    const chargeObj = {
-      amount: "",
-      currency: "",
-      card: {},
-      capture: true
-    };
+    // check if this is a seller shop for destination and transaction fee logic
+    const sellerShop = sellerShops.findOne(paymentData.shopId);
```

Comment:
Can't see an import for `SellerShops` at the top of the file. Looks like it should be 
```js
SellerShops.findOne(paymentData.shopId);
```